oberoncms/adapter-pgsql/src/db/env-config.ts

```diff
@@ -0,0 +1,4 @@
+import { loadEnvConfig } from "@next/env"
```

Comment:
This is vercel specific. My pref is to use regular dotenv where vercel is not a dependency already.